client/components/proposal/ProposalTable.tsx

```diff
@@ -30,58 +37,51 @@ export const ProposalTable = ({ proposalData }) => {
         <h3 className="mt-2 text-sm font-medium text-gray-900">
           No proposals have been created
         </h3>
-        <div className="mt-6">
-          <Link href="/proposal/new" passHref>
-            <a className="btn btn-primary btn-sm">
-              <svg
-                className="-ml-1 mr-2 h-5 w-5"
-                xmlns="http://www.w3.org/2000/svg"
-                viewBox="0 0 20 20"
-                fill="currentColor"
-                aria-hidden="true"
-              >
-                <path
-                  fillRule="evenodd"
-                  clipRule="evenodd"
-                  d="M10 3a1 1 0 011 1v5h5a1 1 0 110 2h-5v5a1 1 0 11-2 0v-5H4a1 1 0 110-2h5V4a1 1 0 011-1z"
-                />
-              </svg>
-              Create Proposal
-            </a>
-          </Link>
-        </div>
       </div>
     );
   }
 
   return (
     <div className="overflow-x-auto">
-      <table className="table w-full">
-        <thead>
-          <tr>
-            <th>#</th>
-            <th>Proposer</th>
-            <th>State</th>
-          </tr>
-        </thead>
+      <table className="table table-fixed w-full">
         <tbody>
-          {proposalData?.proposals.map((proposal, index) => (
-            <tr
-              key={index}
-              className="hover cursor-pointer"
-              onClick={() => router.push(`/proposal/${proposal[0]}`)}
-            >
-              <td>{proposal.displayId}</td>
-              <td>
-                <Address address={proposal[1]} />
-              </td>
-              <td>
-                <StateTag state={proposalData.states[index]} />
-              </td>
-            </tr>
-          ))}
+          {proposalData?.proposals.map((proposal, index) => {
+            const { cleanTitle } = getCleanProposalContent(
+              proposal.description
+            );
+            const id = proposal?.displayId.toString().padStart(3, "0");
+            const lastTx =
+              proposal?.transactions[proposal?.transactions.length - 1];
+
+            return (
+              <tr
+                key={index}
+                className="cursor-pointer hover:bg-gray-100"
+                onClick={() => router.push(`/proposals/${proposal[0]}`)}
```

Comment:
proposal is treated as an object above and as an array here, guess it might be a bug

Reply:
It's actually still an object because it has a `0` key so it's calling in alternative notation. I can see how this is confusing to read though, so I've made it clearer that it's an object using bracket notation.